web-client/src/components/task_creation_form/TaskForm.vue

```diff
@@ -254,14 +248,14 @@ const onSubmit = async (rawTask: any, { resetForm }): Promise<void> => {
 
   // format the flat form entries to a nested task object
   const specialSections = [
-    'generalInformation', 'modelCompileData', 'privacyParamters', 'modelFiles'
```

Comment:
Why are you removing the compilation parameters?

Reply:
it looked to me to be very model specific (and not fit for GPT for eg), I inlined it inside `getModel`. I don't see it as decoupled; and it is needed to compile the model with it to be able to train so expose a `Model.compile` which looked weird.

I'm trying to untangle this omninous global object that `Task` is, cutting fields, not depending on it where I can. I think that the model specific fields are not fit to be there at all, it should be only in `getModel`. but at some point, we will need to discuss a bit more on what we want from that (the no-code platform is an illusion and not useful IMO)